modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -92,8 +92,10 @@ You would want the compatibility setting to accommodate adding new fields withou
 
 == Suggested reading
 
-* xref:reference:console/record-deserialization.adoc[]
 * xref:api:ROOT:pandaproxy-schema-registry.adoc[Schema Registry API]
+* xref:ROOT:reference:console/record-deserialization.adoc[Deserialization]
```

Comment:
The consensus was to avoid sending users to Redpanda docs from the cloud docs. So, we should single-source this page and avoid specifying the `ROOT` component in links.